bin/near-cli.js

```diff
@@ -184,7 +184,8 @@ yargs // eslint-disable-line
     .middleware(require('../middleware/print-options'))
     .middleware(require('../middleware/key-store'))
     .middleware(require('../middleware/ledger'))
-    .command(require('../commands/create-account'))
+    .command(require('../commands/create-account').createAccountCommand)
+    .command(require('../commands/create-account').createAccountCommandDeprecated)
```

Comment:
I'd rather stick with the convention in this file of using the default export. Now that we have two, I would expect the default export to be the correct one. Maybe we could export the deprecated one as a named export, that way when we remove it, the interface will be parallel with all the others here. That would make this file look like this:

```js
    .command(require('../commands/create-account'))
    .command(require('../commands/create-account').deprecated)
```

Reply:
It's probably not worth the while to spend time on this for temporary code.